docs/user-manual/framework/dynamic-memory.md

```diff
@@ -263,20 +266,52 @@ Buffers bins should be tailored based on expected usage. If many small requests
 The above trivial example allows for a few small allocations and one large allocation. In this case, there is a risk that the large allocation is used for the small allocation use case and thus care should be taken to ensure that the smaller use
 cases have a sufficient number of buffers to prevent stealing of larger allocations.
 
-## Separation of `Fw::Buffer` Allocation and Deallocation
+## Deallocation strategy
 
-There is no requirement that the allocating component and the deallocating component are the same. Thus components may
-be chained together for multiple processing steps before deallocation. There are two requirements:
+While there is no requirement that the allocating component and the deallocating component are the same; it is a recommended practice. There are two hard requirements:
```

Comment:
Don't say "while there is no requirement." State "it is recommended practice that".,